### ISSUE TITLE
Cancel button redirects to another page - and this makes the user experience a bit tiresome

### ISSUE THREAD

**Issue by ilias-ant** [scrapy/scrapyd#436]
**Problem**

When someone uses the `Cancel` button(s) available in the `/jobs` page:

![image](https://user-images.githubusercontent.com/42189572/159861556-737921d5-aa31-46fe-8f6e-d5c5ff3e5de4.png)

the action redirects you to the `/cancel.json` page (which is the respective web service response), detailing something like the following:

```
{"node_name": "foobar", "status": "ok", "prevstate": "pending"}
```

If someone wants to perform multiple `Cancel` actions, it has to navigate between the `/jobs` page and the individual redirected `/cancel.json` pages, which makes it a bit tiresome.

**Proposal**

The `Cancel` action through the `/jobs` UI should not redirect, but rather display the response in the same view (`/jobs`). Perhaps this cannot be done purely with HTML, and a bit of Javascript may be needed.

**Comment by jpmckinney** [comment]
Yes, we can maybe make the request to cancel.json and then display the results.

There was a start at a new UI in #304 from before I became maintainer.
